packages/console/compile.tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "composite": true,
     "rootDir": "src",
-    "outDir": "lib"
+    "outDir": "lib",
+    "esModuleInterop": true
```

Comment:
Why is the following change necessary?

Reply:
TSC complains that you can't use default imports without this enabled.